addons/dockable_container/dockable_container.gd

```diff
@@ -334,37 +343,37 @@ func _calculate_panel_and_split_list(result: Array, layout_node: Layout.LayoutNo
 
 
 func _fit_panel_and_split_list_to_rect(panel_and_split_list: Array, rect: Rect2) -> void:
-	"""
-	Traverse list from back to front fitting controls where they belong.
 	
-	Be sure to call this with the result from `_calculate_split_minimum_sizes`.
-	"""
+	# Traverse list from back to front fitting controls where they belong.
+	# 
+	# Be sure to call this with the result from `_calculate_split_minimum_sizes`.
+	
 	var control = panel_and_split_list.pop_back()
 	if control is DockablePanel:
 		_panel_container.fit_child_in_rect(control, rect)
 	elif control is SplitHandle:
 		var split_rects = control.get_split_rects(rect)
-		_split_container.fit_child_in_rect(control, split_rects.self)
+		_split_container.fit_child_in_rect(control, split_rects) # REMOVETHIS original: _split_container.fit_child_in_rect(control, split_rects.self)
```

Comment:
We need to pass `split_rects.self` instead of `split_rects` here. Since we can't write `split_rects.self` directly, just change it to `split_rects['self']`.
In this case, to maintain the Dictionary indexing code style, the lines below could be changed as well to use `split_rects['first']` and `split_rects['second']`.